Modules/Editor/TimedSequenceEditor/TimedSequenceEditor.csproj

```diff
@@ -385,7 +385,7 @@
       <ExcludeAssets>runtime</ExcludeAssets>
       <IncludeAssets>compile; build; native; contentfiles; analyzers; buildtransitive</IncludeAssets>
     </PackageReference>
-    <PackageReference Include="Catel.Serialization.Json" Version="5.12.22">
+    <PackageReference Include="Catel.Serialization.Json" Version="5.12.2">

```

Comment:
Should this be 5.12.10 like the others?

Reply:
Well what I have been testing with is still at 5.12.2.   None of the Wizard stuff shows that as a dependency on Nuget.  Maybe you know the history of when it was added to Vixen and what might need to be spot tested?  We might be able to take that one to 5.12.22.  What would you like me to do?

Reply:
I think it would be best to keep all the Catel libraries in sync at the same version. I have always upgraded them across the board to the same versions to help ensure compatibility.

Reply:
Updated to 5.12.10.